crates/biome_aria/src/roles.rs

```diff
@@ -1241,6 +1241,60 @@ impl<'a> AriaRoles {
 
         false
     }
+
+    pub fn get_corresponding_element(&self, role: &str) -> Option<Iter<(&str, &[(&str, &str)])>> {
```

Comment:
Please document this function, it's a public API.

I remind you Biome philosophy: https://biomejs.dev/internals/philosophy/#technical

> Document Code. Strive to document the code as much as possible, especially the “hard-to-read” code, or for special logic that requires explanation. A well documented code helps its maintainability, especially when multiple people work on it. The developer after you will benefit from your knowledge, share it.